src/main/channels/audio/PostAudiosMessageChannel.ts

```
@@ -0,0 +1,118 @@
+import { IpcMainEvent } from 'electron';
+import log from 'electron-log/main';
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { PostManyAudiosParams } from '../../../shared/interfaces/Audio';
+import { IpcRequest } from '../../../shared/interfaces/IpcRequest';
+import { AppDataSource } from '../../data-source';
+import { Audio } from '../../entity/Audio';
+import { IpcChannel } from '../../interfaces/IpcChannel';
+
+function moveAudioFile(audioId: string, messageId: string) {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)